src/shared/telemetry/defaultTelemetryService.ts

```diff
@@ -293,12 +313,18 @@ export class DefaultTelemetryService implements TelemetryService {
             if (metric.Passive || (didReload && maybeActiveOnReload.includes(metric.MetricName))) {
                 continue
             }
-            const msg = `non-passive metric detected in queue: ${metric.MetricName}`
-            if (isReleaseVersion()) {
-                getLogger().error(msg)
-            } else {
-                throw Error(msg)
-            }
+            this.hasAssertedPassiveStartup = true
```

Comment:
why do this in the for-loop? I also see it below, outside of the for-loop.

Reply:
Miss on my end, was originally `break`ing there and then I realized the point was to list all of the bad telemetry and not stop at one (...although the dev version only alerts one at a time since it throws... 🤔 )